source/includes/code-examples/Restaurant.cs

```diff
@@ -0,0 +1,17 @@
+public class Restaurant
```

Comment:
Suggestion: Consider removing fields of this document that are not used in examples to reduce vertical scroll. I think `Borough` and `Grades` could be good candidates for removal.